README.md

```diff
@@ -16,7 +16,7 @@ provide their own model implementations and use GraphStorm training pipeline to
 
 ## Get Started
 ### Installation
-GraphStorm is compatible to Python 3.7+. It requires PyTorch 1.13+, DGL 1.0 and transformers 4.3.0+.
+GraphStorm is compatible to Python 3.8+. It requires PyTorch 1.13+, DGL 1.0 and transformers 4.3.0+.
```

Comment:
One additional question for all the guys: Should we also move the Pytorch and DGL version?

Reply:
I think it is fine.